### PR TITLE
Add derived properties cache: false example

### PR DESCRIPTION
**Pull request by lulessa**
Plus minor text/code adjustments to clarify use of derived properties.

Please review to be sure reworked language regarding `cache: false` is congruent with AmpersandJS actual behavior.

### CODE REVIEW

File: learn_markdown/state.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ var data = new MyGraphDataModel({numberOfTweets: 555});
 
 // start adding 'em
 var increment = function () {
-    data.number += 1;
+    data.numberOfTweets += 1;
```
Comment (lulessa) [inline learn_markdown/state.md:212]: Seems like `numberOfTweets` is the correct property to increment.

**Comment by pgilad** [comment]
+1

File: learn_markdown/state.md [hunk 2]
```diff
@@ -224,8 +224,25 @@ data.on('change:thousandTweets', function () {
 
 Say you want to calculate a value whenever it's accessed. Sure, you can create a non-cached derived property.
 
-If you say `cache: false` then it will fire a `change` event anytime any of the `deps` changes and it will be re-calculated each time its accessed.
+Set `cache: false` in the derived property object to fire a `change` event anytime any of the `deps` changes and re-calculate the derived property each time it is accessed.
```
Comment (wraithgar) [inline learn_markdown/state.md:227]: any time is two words

**Comment by wraithgar** [comment]
+1 with minor missing space.

**Pull request merged**